articles/quantum-concepts-4-Qubit.md

```diff
@@ -15,67 +15,79 @@ ms.topic: article
 ---
 
 # The Qubit
-Just as bits are the fundamental object of information in classical computing, *qubits* (quantum bits) are the fundamental object of information in quantum computing.  To understand this correspondence, let's look at the simplest example: a single qubit. 
+Just as bits are the fundamental object of information in classical computing, [*qubits*](https://en.wikipedia.org/wiki/Qubit) (quantum bits) are the fundamental object of information in quantum computing.  To understand this correspondence, let's look at the simplest example: a single qubit.
 
-## Representing a Qubit ##
+## Representing a Qubit
+
+While a bit, or binary digit, can have value either $0$ or $1$, a qubit can have a value that is either of these or [a quantum superposition](https://en.wikipedia.org/wiki/Quantum_superposition) of $0$ and $1$. Quantum superposition is the phenomenon where a quantum object can exist across all of its possible states simultaneously. Superposed states are continuous, unlike the discrete classical states $0$ and $1$. When we observe (measure) a qubit in superposition, we collapse its state and are left with only one of our classical states $0$ or $1$.
```

Comment:
I'm not sure the addition here is entirely accurate. Quantum object doesn't exist in all possible states simultaneously, it exists in one specific state which is neither 0 nor 1. And measurement result depends on the basis in which we perform the measurement, but also the result of the collapse is still a quantum state (I remember somebody was confused about how can we apply gates to qubit after measurement if it collapsed to a classical bit). I would keep this paragraph as it was.
@cgranade what do you think?

Reply:
Yeah fair enough, makes sense! It's not strictly correct :)